JudoPayDotNetIntegrationTests/PaymentTest.cs

```diff
@@ -10,22 +11,26 @@ namespace JudoPayDotNetIntegrationTests
     public class PaymentTest
     {
         private JudoPayApi _judo;
+        private readonly Configuration _configuration = new Configuration();
+
+        [SetUp]
+        public void SetUp()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;
```

Comment:
I believe until the migration is complete this can stay the same as per our code base. E.g. D:\Projects\Platform\Code\Judo.Web.Partner.Api\Global.asax.cs Line 54